### DISCUSSION TITLE
커밋을 삭제했는데 contribution 기록이 삭제가 안됨

### DISCUSSION THREAD

**Discussion by qkrwldns** [community/community#140744] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

커밋을 삭제했는데 contribution 기록이 삭제가 안됨

**Comment by ebndev** [comment]
Hi @qkrwldns, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
